t_ja5/test_ja5t.py

```diff
@@ -184,15 +186,18 @@ def change_cfg(self):
         tempesta: Tempesta = self.get_tempesta()
         config = tempesta.config.defconfig
         ja5_configs = [
+            self.tempesta_ja5_config_empty,
             self.tempesta_ja5_config_1,
             self.tempesta_ja5_config_2,
-            self.tempesta_ja5_config_empty,
         ] * 4
 
         for ja5_config in ja5_configs:
             tempesta.config.defconfig = config + ja5_config
             self.get_tempesta().reload()
 
+            # sleep sometime to receive 200 responses
+            time.sleep(0.5)
```

Comment:
I think we can disable this test on TCP seg because we use wrk and h2load clients here (they doesn't work with TCP seg) and ja5 which only works with requests. Please remove this sleep and add this tests to `tests_dsaibled_tcpseg,json` woth reason - "This test should not be run with TCP segmentation. It use h2load and wrk clients and ja5 only works for requests."